com.chartboost.mediation/Android~/chartboost-mediation-android-bridge/src/main/java/com/chartboost/mediation/unity/AdWrapper.kt

```diff
@@ -157,13 +157,10 @@ class AdWrapper(private val ad: HeliumAd) {
         // Attach the banner layout to the activity.
         val density = displayDensity
         try {
-
-            val size = ad.getSize()
-
-            when {
-                size == HeliumBannerSize.LEADERBOARD -> ad.layoutParams = getBannerLayoutParams(density, LEADERBOARD.first, LEADERBOARD.second)
-                size == HeliumBannerSize.MEDIUM -> ad.layoutParams = getBannerLayoutParams(density, MEDIUM.first, MEDIUM.second)
-                else -> ad.layoutParams = getBannerLayoutParams(density, STANDARD.first, STANDARD.second)
+            when (ad.getSize()?.name ?: "STANDARD") {
+                "LEADERBOARD" -> ad.layoutParams = getBannerLayoutParams(density, LEADERBOARD.first, LEADERBOARD.second)
+                "MEDIUM" -> ad.layoutParams = getBannerLayoutParams(density, MEDIUM.first, MEDIUM.second)
+                "STANDARD" -> ad.layoutParams = getBannerLayoutParams(density, STANDARD.first, STANDARD.second)
```

Comment:
Is this based on what was done last release? 🤔

Reply:
@cb-jpadilla Yes, this seems to be a modification to such change.